pre_commit_mirror_maker/all/.pre-commit-hooks.yaml

```diff
@@ -4,3 +4,4 @@
     language: {language}
     '{match_key}': {match_val}
     args: {args}
+    additional_dependencies: {additional_dependencies}
```

Comment:
How important is the `minimum_pre_commit_version` key? Is it worth doing some hackery to get it in for projects which have `additional_dependencies`?

Reply:
probably not all that important,  it's not exactly a _new_ feature (and will fail for other reasons) 😆